go.mod

```diff
@@ -2,34 +2,20 @@ module github.com/Yelp/paasta-tools-go
 
 require (
 	github.com/dlespiau/kube-test-harness v0.0.0-20200730130322-72c5b0037f4a
-	github.com/fatih/structs v1.1.0
-	github.com/go-logr/zapr v0.1.1 // indirect
-	github.com/go-openapi/errors v0.19.4
-	github.com/go-openapi/runtime v0.19.15
-	github.com/go-openapi/strfmt v0.19.5
-	github.com/go-openapi/swag v0.19.5
-	github.com/go-openapi/validate v0.19.8
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/golang/groupcache v0.0.0-20191227052852-215e87163ea7 // indirect
-	github.com/hashicorp/golang-lru v0.5.3 // indirect
-	github.com/mitchellh/mapstructure v1.2.2
-	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826
+	github.com/golang/glog v1.0.0
+	github.com/mitchellh/mapstructure v1.4.1
 	github.com/openzipkin/zipkin-go v0.2.2
-	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
-	github.com/prometheus/client_golang v1.3.0 // indirect
-	github.com/stretchr/testify v1.6.1
+	github.com/stretchr/testify v1.8.1
 	github.com/subosito/gotenv v1.2.0
-	go.uber.org/zap v1.13.0 // indirect
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v0.18.6
+	github.yelpcorp.com/go-packages/monk v0.0.0-20210923142653-46be63e1f16e
+	golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.26.2
+	k8s.io/apimachinery v0.26.2
+	k8s.io/client-go v0.26.2
 	k8s.io/klog v1.0.0
-	k8s.io/kubernetes v1.14.0
-	sigs.k8s.io/controller-runtime v0.6.0
-	sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e // indirect
+	sigs.k8s.io/controller-runtime v0.14.5
 )
 
 go 1.12
```

Comment:
go 1.12 is very old, can we update it?
i think 1.17 should be safe since we have that on all bionic hosts and in any Jenkins pipeline.
on Jammy we already have 1.20.
so i'm inclined to test 1.20 first and downgrade to 1.17 if necessary